bgpd/bgp_evpn_vty.c

```diff
@@ -2655,6 +2673,33 @@ DEFUN (no_bgp_evpn_advertise_all_vni,
 	return CMD_SUCCESS;
 }
 
+DEFUN (bgp_evpn_advertise_autort_rfc8365,
+       bgp_evpn_advertise_autort_rfc8365_cmd,
+       "advertise-autort-rfc8365",
```

Comment:
I have a few comments/suggestions here:
0. Can we think about making this the default behaviour and remove this command ? It will be good to know if all major vendors adhere to this format.
1. could we please change this command to be something more user friendly instead of referring to a specific RFC?

Reply:
For 0, this would break existing setups, notably for your customers. But the patch would be far simpler if there was no option.

For 1, open to suggestions.

Reply:
I think it will be okay to make this the default encoding. this will create a problem only when different switches run different versions of FRR (might not be that common).
Lets wait for @vivek-cumulus  inputs.

Reply:
Customers upgrading from one FRR version to a new one will break their whole setup.

Reply:
Yes, you are right.